### PR TITLE
[HOTFIX] GPUSolver Type Checking

### PR DESCRIPTION
**Pull request by wbinventor**
This PR fixes some type checking for `Solver` classes in the `openmoc.process` and `openmoc.plotter` modules. Now one can extract, plot and store fluxes from a `GPUSolver` without one of those modules throwing an error in the new type checking interface.

### CODE REVIEW

File: openmoc/plotter.py [hunk 1]
```diff
@@ -723,7 +736,9 @@ def plot_energy_fluxes(solver, fsrs, group_bounds=None, norm=True,
 
     """
 
-    cv.check_type('solver', solver, openmoc.Solver)
+    global solver_types
+    cv.check_type('solver', solver, solver_types)
+#    cv.check_type('solver', solver, (openmoc.Solver, openmoc.cuda.GPUSolver))
```
Comment (geogunow) [inline openmoc/plotter.py]: Remove

File: openmoc/plotter.py [hunk 2]
```diff
@@ -865,7 +880,9 @@ def plot_fission_rates(solver, norm=False, transparent_zeros=True, gridsize=250,
...
```
Comment (geogunow) [inline openmoc/plotter.py]: Remove

File: openmoc/plotter.py [hunk 3]
```diff
@@ -1216,7 +1233,9 @@ def plot_quadrature(solver, get_figure=False):
...
```
Comment (geogunow) [inline openmoc/plotter.py]: Remove

**Comment by wbinventor** [comment]
@geogunow I addressed your comments.

**Pull request merged**